addons/l10n_it_edi/tests/IT01234567890_FPR01.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could you put the file in a different directory `test_file` ?

Reply:
Call it expected_xml(s) like in l10n_cl_edi then

Reply:
Ok!